packages/frontend-2/components/header/NavBar.vue

```diff
@@ -1,35 +1,36 @@
 <template>
-  <Disclosure v-slot="{ open }" as="nav" class="bg-background shadow-sm">
-    <div class="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
-      <div class="flex h-16 justify-between">
+  <Disclosure
+    v-slot="{ open }"
+    as="nav"
+    class="group bg-background shadow-md hover:shadow-lg transition fixed w-full z-10"
+  >
+    <div class="mx-auto px-4 default-width">
+      <div class="flex h-14 group-hover:h-16 transition-all justify-between">
         <div class="flex">
-          <div class="flex flex-shrink-0 items-center">
-            <HeaderLogoBlock minimal />
-          </div>
-          <div class="hidden sm:-my-px sm:ml-6 sm:flex sm:space-x-8">
-            <a
-              v-for="item in navigation"
-              :key="item.name"
-              :href="item.href"
-              :class="[
-                item.current
-                  ? 'border-primary text-foreground'
-                  : 'border-transparent text-foreground-3 hover:text-foreground-2 hover:border-foreground-4',
-                'inline-flex items-center px-1 pt-1 border-b-2 text-sm font-medium'
-              ]"
-              :aria-current="item.current ? 'page' : undefined"
-            >
-              {{ item.name }}
-            </a>
+          <HeaderLogoBlock :active="false" class="mr-1" />
+          <div class="hidden sm:flex flex-shrink-0 items-center">
+            <HeaderNavLink to="/" name="Dashboard" :separator="false" class="ml-2" />
+
+            <HeaderNavLink
+              v-for="(nl, i) in nav.filter((n) => !!n)"
```

Comment:
these probably can't be null, and if they can, better to clean up `nav` (or create a computed?) instead of introducing filtering logic here in the template

Reply:
YOLO, we use your system (once you code it up!)